microsoft-edge/extensions-chromium/getting-started/part2-content-scripts.md

```diff
@@ -228,100 +409,47 @@ The updated `manifest.json` that includes the `content-scripts` and `web_accessi
 }
 ```
 
-The `matches` attribute is set to `<all_urls>`, which means that all files in `content_scripts` are injected into all browser tab pages when each tab is loaded.  The allowed files types that can be injected are JavaScript and CSS.  You also added `lib\jquery.min.js`.  You can copy that file from the [/lib/](https://github.com/microsoft/MicrosoftEdge-Extensions/tree/main/Extension%20samples/extension-getting-started-part2/extension-getting-started-part2/lib) folder of the MicrosoftEdge-Extensions repo.
-
 
 <!-- ------------------------------ -->
-#### Add jQuery
-
-This step has already been done in [jquery.min.js](https://github.com/microsoft/MicrosoftEdge-Extensions/blob/main/Extension%20samples/extension-getting-started-part2/extension-getting-started-part2/lib/jquery.min.js) in the MicrosoftEdge-Extensions repo.
+#### The manifest's `content_scripts` section injects JavaScript into all browser tabs
 
-In the content scripts that you're injecting, plan on using jQuery (`$`).  You added a minified version of jQuery and put it in your extension package as `lib\jquery.min.js`.
+In the `content_scripts` key section, the `matches` attribute is set to `<all_urls>`, which means that `.js` (or `.css`) files that are specified in In the `content_scripts` key section are injected into all browser tab pages when each tab is loaded.  The allowed files types that can be injected are JavaScript (`.js`) and CSS (`.css`).  
 
-These content scripts run in individual sandboxes, which means that the jQuery injected into the `popup.js` page isn't shared with the content.
+In this sample, the following files are specified:
+* `content-scripts/content.js`
+* `lib/jquery.min.js`
 
 
 <!-- ------------------------------ -->
-#### Understanding the thread
-
-Even if the browser tab has JavaScript running on it on the loaded web page, any content that's injected doesn't have access to that JavaScript.  The injected JavaScript only has access to the actual DOM that's loaded in that browser tab.
-<!-- todo: mention 'thread', since heading says 'thread' -->
+#### How jQuery support is added (`jquery.min.js`)
 
+`jquery.min.js` is a predefined, minified file to support jQuery expressions starting with `$` in `popup.js` and `content.js`.
```

Comment:
JQuery is only used in content.js, not in popup.js.

```suggestion
`jquery.min.js` is a predefined, minified file to support jQuery expressions starting with `$` in `content.js`.
```

Reply:
popup.js contains line that contains `$`:
```
imageDivId: `${guidGenerator()}`,
```
@captainbrosset - keep mention of popup.js?

Reply:
These are 2 different use of `$`:

* `$(...)`
* `${...}`

The former is the jQuery selector function. The latter is a placeholder which can be used in JS template strings.

If popup.js only contains the latter, then it doesn't actually uses jQuery, and therefore doesn't need it.